core/selection.js

```diff
@@ -1018,15 +1027,13 @@
 			}
 
 			function nonEditableAscendant( sel ) {
-				if ( CKEDITOR.env.ie ) {
-					var range = sel.getRanges()[ 0 ],
-						ascendant = range ? range.startContainer.getAscendant( function( parent ) {
-								return parent.type == CKEDITOR.NODE_ELEMENT &&
-									( parent.getAttribute( 'contenteditable' ) == 'false' || parent.getAttribute( 'contenteditable' ) == 'true' );
-							}, true ) : null ;
-
-					return range && ascendant.getAttribute( 'contenteditable' ) == 'false' && ascendant;
-				}
+				var range = sel.getRanges()[ 0 ],
+					ascendant = range ? range.startContainer.getAscendant( function( parent ) {
+						return parent.type == CKEDITOR.NODE_ELEMENT &&
+							( parent.getAttribute( 'contenteditable' ) == 'false' || parent.getAttribute( 'contenteditable' ) == 'true' );
+					}, true ) : null ;
+
+				return range && ascendant.getAttribute( 'contenteditable' ) == 'false' && ascendant;
```

Comment:
I'd recommend revise this line. if 'ascendant' is null, ascendant.getAttribute() should throw exception. truthiness check for 'ascendant' should be done before calling getAttribute.

Reply:
@mokona88 is right – current condition is wrong. Existence of `ascendant` should be checked before checking its attribute.